python/arcticdb/version_store/library.py

```diff
@@ -966,9 +966,9 @@ def append(
         metadata
             Optional metadata to persist along with the new symbol version. Note that the metadata is
             not combined in any way with the metadata stored in the previous version.
-        prune_previous_versions, default=False
```

Comment:
This and `validate_index` were causing `mkdocs serve -s -a 0.0.0.0:8000` to fail in strict mode with 
```
WARNING -  griffe: /home/vasil/Documents/source/ArcticDB/python/arcticdb/version_store/library.py:973: Parameter 'default' does not appear in the function signature
```
mkdocs parses the function signature and can show the default without mentioning it in the docstring.